src/index.html

```diff
@@ -11,12 +11,73 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="MoYo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__menu">
+          <li class="nav__menu__item">
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__menu__item">
+            <a href="#">Samsung</a>
+          </li>
+          <li class="nav__menu__item">
+            <a href="#">Smartphones</a>
+          </li>
+          <li class="nav__menu__item">
+            <a
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__menu__item">
+            <a href="#">Gadgets</a>
+          </li>
+          <li class="nav__menu__item">
+            <a href="#">Tablets</a>
+          </li>
+          <li class="nav__menu__item">
+            <a href="#">Photo</a>
+          </li>
+          <li class="nav__menu__item">
+            <a href="#">Video</a>
```

Comment:
Using '#' as the href value is not recommended for accessibility and SEO. Consider using a more descriptive URL or a JavaScript function to handle the click event if it's meant to be a placeholder.